### PR TITLE
added slow query

### PR DESCRIPTION
**Pull request by cooper-lzy**

### CODE REVIEW

File: docs-2.0/5.configurations-and-logs/1.configurations/4.storage-config.md [hunk 1]
```diff
@@ -74,9 +74,10 @@ Storage服务提供了两份初始配置文件`nebula-storaged.conf.default`和`
 
 ## disk配置
 
-| 名称                       | 默认值            | 说明                     |
+| 名称                       | 预设值            | 说明                     |
 | :--------------------------| :-------------- | :------------------------ |
 |`data_path`                 | `data/storage`  | 数据存储路径，多个路径用英文逗号（,）分隔。一个RocksDB实例对应一个路径。  |
+|`minimum_reserved_bytes`|`268435456`|每个数据存储路径的剩余空间最小值，低于该值时，可能会导致集群写入失败。单位：字节。默认为`1073741824`，即1GB。|
```
Comment (whitewum) [inline docs-2.0/5.configurations-and-logs/1.configurations/4.storage-config.md:80]: 256？

File: docs-2.0/5.configurations-and-logs/1.configurations/2.meta-config.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ Meta服务提供了两份初始配置文件`nebula-metad.conf.default`和`nebula
 | `pid_file`  | `pids/nebula-metad.pid` | 记录进程ID的文件。   |
 | `timezone_name` | - | 指定Nebula Graph的时区。初始配置文件中未设置该参数，如需使用请手动添加。系统默认值为`UTC+00:00:00`。格式请参见[Specifying the Time Zone with TZ](https://www.gnu.org/software/libc/manual/html_node/TZ-Variable.html "Click to view the timezone-related content in the GNU C Library manual")。例如，东八区的设置方式为`--timezone_name=CST-8`。 |
 | `local_config` | `true` | 是否从配置文件获取配置信息。 |
+|`minimum_reserved_bytes`|-|每个数据存储路径的剩余空间最小值，低于该值时，可能会导致集群写入失败。单位：字节。默认为`1073741824`，即1GB。|
```
Comment (whitewum) [inline docs-2.0/5.configurations-and-logs/1.configurations/2.meta-config.md:28]: 256？

Comment (whitewum) [reply docs-2.0/5.configurations-and-logs/1.configurations/2.meta-config.md:28]: 可能会导致集群元数据操作失败

**Review by whitewum — COMMENTED** [review]
https://docs.nebula-graph.com.cn/master/20.appendix/0.FAQ/#_5
don't forget this

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/18.show-querise.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+`SHOW QUERIES`语句可以查看当前Session中正在执行的查询请求信息。
+
+!!! note
+
+    如果需要终止查询，请参见[终止查询](../../18.operation-and-maintenance-statements/6.kill-query.md)。
+
+## 注意事项
+
+- `SHOW QUERIES`查询当前Session的查询请求，因为信息存储在本地缓存，所以几乎没有延迟。
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/18.show-querise.md:11]:
```suggestion
- `SHOW QUERIES`从本地缓存获取当前Session中查询的状态，几乎没有延迟。
```

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/18.show-querise.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+- `SHOW ALL QUERIES`查询所有Session的查询请求，需要从Meta服务获取，Session的信息会定时（参数`session_reclaim_interval_secs`）发送给Meta，因此存在延迟。
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/18.show-querise.md:13]:
```suggestion
- `SHOW ALL QUERIES`从Meta服务获取所有Session中的查询信息。这些信息会根据参数`session_reclaim_interval_secs`定义的周期同步到Meta服务，因此在客户端获取到的信息可能属于上个同步周期。
```

File: docs-2.0/5.configurations-and-logs/1.configurations/2.meta-config.md [hunk 5]
```diff
@@ -25,6 +25,7 @@ Meta服务提供了两份初始配置文件`nebula-metad.conf.default`和`nebula
 | `pid_file`  | `pids/nebula-metad.pid` | 记录进程ID的文件。   |
 | `timezone_name` | - | 指定Nebula Graph的时区。初始配置文件中未设置该参数，如需使用请手动添加。系统默认值为`UTC+00:00:00`。格式请参见[Specifying the Time Zone with TZ](https://www.gnu.org/software/libc/manual/html_node/TZ-Variable.html "Click to view the timezone-related content in the GNU C Library manual")。例如，东八区的设置方式为`--timezone_name=CST-8`。 |
 | `local_config` | `true` | 是否从配置文件获取配置信息。 |
+|`minimum_reserved_bytes`|-|每个数据存储路径的剩余空间最小值，低于该值时，可能会导致集群元数据操作失败。单位：字节。默认为`1073741824`，即1GB。|
```
Comment (randomJoe211) [inline docs-2.0/5.configurations-and-logs/1.configurations/2.meta-config.md:28]: 要不要在下面加个Caution或者Danger?

**Review by randomJoe211 — DISMISSED** [review]

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/18.show-querise.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+## 注意事项
+
+- `SHOW QUERIES`从本地缓存获取当前Session中查询的状态，几乎没有延迟。
+
+- `SHOW ALL QUERIES`从Meta服务获取所有Session中的查询信息。这些信息会根据参数`session_reclaim_interval_secs`定义的周期同步到Meta服务，因此在客户端获取到的信息可能属于上个同步周期。
+
+## 语法
+
+```ngql
+SHOW QUERIES;
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/18.show-querise.md:18]: you can also write as 
```
SHOW [ALL] QUERIES;
```

**Review by randomJoe211 — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**